README.md

```diff
@@ -18,7 +18,7 @@ Here is how to install it:
 
 Add the repo key:
 
-    $ wget -O - https://facebook.github.io/mcrouter/debrepo/xenial/PUBLIC.KEY | sudo apt-key add
+    $ wget -qO- https://facebook.github.io/mcrouter/debrepo/xenial/PUBLIC.KEY | sudo apt-key add
```

Comment:
I agree with this, but maybe not related enough to the goal of this PR to include.